Vagrantfile

```diff
@@ -24,7 +24,7 @@ Vagrant.configure("2") do |config|
   # Create a forwarded port mapping which allows access to a specific port
   # within the machine from a port on the host machine. In the example below,
   # accessing "localhost:8080" will access port 80 on the guest machine.
-  config.vm.network "forwarded_port", guest: 80, host: 2080
+  config.vm.network "forwarded_port", guest: 80, host: 8080
```

Comment:
i changed this to 8080, that was ok, right?

Reply:
yea if that's what you want.  This is what homestead did.